docs/data/data-grid/demo/PopularFeaturesDemo.js

```diff
@@ -231,38 +233,71 @@ export const featuresSet = [
 function getChipProperties(plan) {
   switch (plan) {
     case 'Premium':
-      return { avatarLink: '/static/x/premium.svg', color: '#ffecc8' };
+      return { avatarLink: '/static/x/premium.svg' };
     case 'Pro':
-      return { avatarLink: '/static/x/pro.svg', color: '#c8e9ff' };
+      return { avatarLink: '/static/x/pro.svg' };
     default:
-      return { avatarLink: undefined, color: '#c8ffdb' };
+      return { avatarLink: '/static/x/community.svg' };
   }
 }
 
 function PlanTag(props) {
-  const chipPropperties = getChipProperties(props.plan);
-  const avatar = !chipPropperties.avatarLink ? undefined : (
-    <img src={chipPropperties.avatarLink} width={21} height={24} alt="" />
+  const theme = useTheme();
+  const chipProperties = getChipProperties(props.plan);
+  const avatar = !chipProperties.avatarLink ? undefined : (
+    <img src={chipProperties.avatarLink} width={21} height={24} alt="" />
   );
 
   return (
     <Chip
+      variant="outlined"
+      size="small"
       avatar={avatar}
+      label={props.plan}
       sx={{
-        backgroundColor: chipPropperties.color,
-        color: 'rgba(0, 0, 0, 0.87)',
+        pl: 0.5,
+        ...(props.plan === 'Premium' && {
+          backgroundColor:
+            theme.palette.mode === 'dark' ? alpha(yellow[900], 0.4) : yellow[50],
+          borderColor:
+            theme.palette.mode === 'dark'
+              ? alpha(yellow[300], 0.4)
+              : alpha(yellow[900], 0.4),
+        }),
+        ...(props.plan === 'Pro' && {
+          backgroundColor:
+            theme.palette.mode === 'dark' ? alpha(blue[600], 0.4) : blue[50],
+          borderColor:
+            theme.palette.mode === 'dark'
+              ? alpha(blue[300], 0.4)
+              : alpha(blue[900], 0.2),
+        }),
+        ...(props.plan === 'Community' && {
+          backgroundColor:
+            theme.palette.mode === 'dark' ? alpha(green[600], 0.4) : green[50],
+          borderColor:
+            theme.palette.mode === 'dark'
+              ? alpha(green[300], 0.4)
+              : alpha(green[900], 0.2),
+        }),
+        '& .MuiChip-label': {
+          fontWeight: 'medium',
+          fontSize: theme.typography.pxToRem(12),
+          pl: 1.5,
```

Comment:
```suggestion
          pl: 1,
```

This padding looks a bit too large:


https://github.com/mui/mui-x/assets/13808724/efc4e1f4-ac4f-4fcd-98d7-567c9440c97a